src/components/picker/nimble-picker.js

```diff
@@ -278,10 +284,161 @@ export default class NimblePicker extends React.PureComponent {
     }
 
     this.leaveTimeout = setTimeout(() => {
-      preview.setState({ emoji: null })
+      this.setState({ emoji: null })
     }, 16)
   }
 
+  handleEmojiKeyDown(e, _, { category, row, column }) {
+    const { perLine } = this.props
+    const categoryIndex = this.categories.findIndex(({ id }) => id === category)
+
+    const getEmojiInCategory = (categoryIndex) => {
+      const emojisInCategory =
+        categoryIndex === 1
+          ? frequently.get(perLine)
+          : this.categories[categoryIndex].emojis
+      return emojisInCategory
+    }
+
+    const getLastEmojiIndex = (categoryIndex) => {
+      const emojisInCategory = getEmojiInCategory(categoryIndex)
+      const lastEmojiIndex = emojisInCategory.length - 1
+      return lastEmojiIndex
+    }
+
+    let newRow
+    let newColumn
+    let newCategoryIndex = categoryIndex
+    let emojiIndex
+    const lastEmojiIndex = getLastEmojiIndex(categoryIndex)
+
+    switch (e.keyCode) {
```

Comment:
> This feature is no longer recommended.
> […]
> You should avoid using this if possible; it's been deprecated for some time. Instead, you should use KeyboardEvent.code, if it's implemented. Unfortunately, some browsers still don't have it, so you'll have to be careful to make sure you use one which is supported on all target browsers.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Could we try not introducing legacy code here, possibly?